llvm/utils/git/github-automation.py

```diff
@@ -241,6 +250,58 @@ def run(self) -> bool:
         return True
 
 
+class CommitRequestGreeter:
+    def __init__(self, token: str, repo: str, issue_number: int):
+        self.repo = github.Github(token).get_repo(repo)
+        self.issue = self.repo.get_issue(issue_number)
+
+    def run(self) -> bool:
+        # Post greeter comment:
+        comment = f"""
+            @{self.issue.user.login} thank you for apply for commit access.  Please  review the project's [code review policy](https://llvm.org/docs/CodeReview.html).
+        """
+        self.issue.create_comment(textwrap.dedent(comment))
```

Comment:
@boomanaiden154 Was this what you were thinking when you suggested using dedent?

Reply:
I typically do this as `comment = textwrap.dedent(f"""<line breaks and text here>""")` and then use `comment` later on without calling `textwrap.dedent` again. I think either way should work though.

Reply:
Ok, I think I this is fixed now.

Reply:
Yeah, LGTM to me.